docs/cmd/changelogutils/changelog_utils.go

```diff
@@ -71,20 +57,32 @@ func ParseReleases(releases []*github.RepositoryRelease, byMinorVersion bool) (m
 // This also pulls in open source gloo edge release notes and merges them with enterprise release notes
 // The returned map will be a mapping of minor releases (v1.5, v1.6) to their body, which will contain the release notes
 // for all the patches under the minor releases
```

Comment:
nit: Should probably update this comment since the func is exported.
```suggestion
// This also pulls in open source gloo edge release notes and merges them with enterprise release notes
// The returned map will be a mapping of minor releases (v1.5, v1.6) to their body, which will contain the release notes
// for all the patches under the minor releases. It also returns a list of versions, sorted by version if the sortedByVersion param is true.
```